src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -19,9 +19,11 @@ const wheelZoomLevel = 0.15;
 const currentAssetClassName = 'current-asset';
 const minDesktopWidth = 992;
 
-export interface SvgViewerProps {
+interface SvgViewerProps {
   // CDF fileId to fetch svg-document
   documentId: number;
+  // Function to replace document fetching call
+  fetchDocument?: (documentId: number) => Promise<string>;
```

Comment:
```suggestion
  fetchDocument<T>?: (...params: T) => Promise<string>;
```

Reply:
`documentId` is something that comes from the inside of component, in this case it is `documentId` prop, which has type `number`

Reply:
right